simplq/src/components/pages/Home/Nav/Navbar.jsx

```diff
@@ -5,7 +5,7 @@ import Burger from './Burger';
 const Navbar = () => {
   return (
     <nav className={styles['navbar']}>
-      <img src="LogoLight.png" />
+      <img src="LogoLight.png" alt="Home" />
```

Comment:
And this change too.

Reply:
I'm not sure what the problem here is, but I just pulled your latest code and the changes are not there.
![Screenshot from 2020-10-01 22-55-05](https://user-images.githubusercontent.com/12583528/94842566-2d9f6580-0439-11eb-8459-85f294ce42c8.png)


![Screenshot from 2020-10-01 22-54-46](https://user-images.githubusercontent.com/12583528/94842586-342ddd00-0439-11eb-863a-0d326228f982.png)

And the diff you're referencing above seems to be outdated.
![Screenshot from 2020-10-01 22-54-03](https://user-images.githubusercontent.com/12583528/94842633-3f810880-0439-11eb-89e8-0399d0d97b7c.png)

Reply:
What is the problem then?
let me push once more, maybe changes will reflect then.

Reply:
From my side, it is showing nothing to commit, and in my files, these changes are already done.
![Screenshot from 2020-10-01 22-59-37](https://user-images.githubusercontent.com/48750169/94843310-2593f580-043a-11eb-9b0d-3a3e945920de.png)
![Screenshot from 2020-10-01 23-01-09](https://user-images.githubusercontent.com/48750169/94843314-275db900-043a-11eb-9843-9e4f82adb97b.png)
![Screenshot from 2020-10-01 23-01-26](https://user-images.githubusercontent.com/48750169/94843319-27f64f80-043a-11eb-93d4-483b26920573.png)

Reply:
The other change I have done i.e width:15rem before this push is reflecting but why not these?

Reply:
I'm looking at your github page and the changes aren't there either https://github.com/arkumari2000/simplQ-frontend/blob/new-frontend/simplq/src/components/pages/Home/Nav/Navbar.jsx. Try pulling from here first, then check and  push. I think it's because @daltonfury42 merged some changes to it which may have accidentally reverted these changes you made.

Reply:
Okay, I will try to do this. maybe things will work fine then.

Reply:
Done.
Please check now.

Reply:
Okay, looks like we're good now. Was that the issue?

Reply:
Yes, I pulled first and then had to make the changes again.
maybe due to merge this issue came.